src/main/java/com/backbase/ct/bbfuel/data/ProductSummaryDataGenerator.java

```diff
@@ -107,6 +107,16 @@ private static String getProductTypeNameFromProductsInputFile(String productId)
         return arrangementsPostRequestBodies;
     }
 
+    public static List<ArrangementsPostRequestBody> generateCurrentAccountArrangementsPostRequestBodiesWithStates
```

Comment:
you don't have to create a new method as you can set the `stateId` in existing method `getArrangementsPostRequestBody` like e.g. `.withStateId(getRandomFromList(arrangementStateExternalIds));`